cursorless-talon/src/actions/actions_makeshift.py

```diff
@@ -39,7 +39,7 @@ class MakeshiftAction:
         "editor.action.rename",
         restore_selection=True,
         await_command=False,
-        post_command_sleep_ms=150,
+        post_command_sleep_ms=200,
```

Comment:
reminder to self to try this out locally

Reply:
200 ms seems to work fine for me atleast